t/cobrand/brent.t

```diff
@@ -102,4 +102,26 @@ subtest "UnitID on gully sent across in detail" => sub {
     };
 };
 
+subtest "Report title passed through as attribute" => sub {
```

Comment:
This is fine, though I probably would have only added the `is $c->param('attribute[title]')` line to the test above :)